src/components/study/display-study-status.test.tsx

```diff
@@ -1,9 +1,14 @@
-import { describe, expect, it } from 'vitest'
+import { describe, expect, it, vi } from 'vitest'
 import { StudyJobStatus, StudyStatus } from '@/database/types'
 import { renderWithProviders } from '@/tests/unit.helpers'
 import { screen } from '@testing-library/react'
 import { DisplayStudyStatus } from './display-study-status'
 
+let mockPathname = '/'
+vi.mock('next/navigation', () => ({
+    usePathname: () => mockPathname,
```

Comment:
we're already mocking useParams from 'next/navigation' in unit.helpers, how about adding this to it